packages/table/src/table.tsx

```diff
@@ -806,10 +849,6 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
      * intend to redraw the region layer.
      */
     private maybeRenderRegions(getRegionStyle: IRegionStyler) {
-        if (this.isGuidesShowing()) {
-            return undefined;
-        }
```

Comment:
well "And then selected regions remain broken even after the reordering completes for some reason." worries me that we don't know why. but okay